tests/foreman/ui/test_subnet.py

```diff
@@ -293,3 +293,376 @@ def test_positive_update_mask(self):
+
+        :expectedresults: The parameters from subnet should not be displayed
+            in host parameters tab
+        """
+
+    @stubbed
+    @tier2
+    def test_positive_subnet_parameters_override_from_host(self):
+        """Subnet parameters value can override from host
+
+        :id: fa8c4a14-4a6e-4116-84db-72d2854858da
+
+        :steps:
+
+            1. Create valid subnet with valid parameter
+            2. Create host with above subnet
+            3. Assign hosts primary interface with subnet
+            4. Go to that host parameters tab and override subnet parameter
+                value
+            5. Submit the override the changes in subnet.
+
+        :expectedresults: The subnet parameters should override from host
+        """
+
+    @stubbed
+    @tier2
+    def test_positive_subnet_parameters_override_impact_on_subnet(self):
+        """Override subnet parameter from host impact on subnet parameter
+
+        :id: 63280dd0-4e29-486b-a5ee-d33d8cc93f3d
+
+        :steps:
+
+            1. Create valid subnet with valid parameter
+            2. Create host with above subnet
+            3. Assign hosts primary interface with subnet
+            4. Go to that host parameters tab and override subnet parameter
+                value
+            5. Submit the override the changes in subnet.
+
+        :expectedresults: The override value of subnet parameter from host
+            should not change actual value in subnet parameter
+        """
+
+    @stubbed
+    @tier1
+    def test_positive_update_subnet_parameter(self):
+        """Subnet parameter can be updated
+
+        :id: d9313a4b-20ae-4bb1-b395-0e2ecbb63397
+
+        :steps:
+
+            1. Create a subnet from Infrastructure menu.
+            2. Update all the details for subnet creation
+            3. Add parameter in subnet parameters tab
+            4. Save subnet changes
+            5. Edit the above subnet and update parameter with new name and
+                value
+            6. Save subnet changes
+
+        :expectedresults: The parameter name and value should be updated
+        """
+
+    @stubbed
+    @tier1
+    def test_negative_update_subnet_parameter(self):
+        """Subnet parameter can not be updated with invalid names
+
+        :id: ef5a8d41-d131-4a43-8a32-01df90334326
+
+        :steps:
+
+            1. Create a subnet from Infrastructure menu.
+            2. Update all the details for subnet creation
+            3. Add parameter in subnet parameters tab
+            4. Save subnet changes
+            5. Edit the above subnet and update parameter with some invalid
+                name. e.g name with comma or space
+            6. Save subnet changes
+
+        :expectedresults: The parameter should not be updated with invalid name
+        """
+
+    @stubbed
+    @tier2
+    def test_positive_update_subnet_parameter_host_impact(self):
+        """Subnet parameter update impact onto host
```

Comment:
Clarify that statement
It is not obvious for me what `update impact onto host` means. How can we update impact?